packages/integration-tests/src/tests/api/experience-api/verifications-api/password-verification.test.ts

```diff
@@ -0,0 +1,43 @@
+import { deleteUser } from '#src/api/admin-user.js';
+import { createPasswordVerification } from '#src/api/experience-api/password-verification.js';
+import { initClient } from '#src/helpers/client.js';
+import { generateNewUser } from '#src/helpers/user.js';
+import { devFeatureTest } from '#src/utils.js';
+
+const signInIdentifiersType: readonly ['username', 'email', 'phone'] = Object.freeze([
+  'username',
+  'email',
+  'phone',
+]);
```

Comment:
If we use enum then this can be removed

Reply:
updated using enum. Was following old implementation.